### PR TITLE
displays workshop action button according to logged in user and workshop status

### LINKED ISSUE #161: Display of workshop action button
In the workshop list page Presently we display 4-5 button depending on status only.
We need to add more logic to show button based on logged in person
1) Hold button should be displayed only if logged-in person is member of workshop requested organisation and workshop is not completed.
2) Requested button should be display only if logged-in person is member of workshop requested organisation and workshop is in hold status.
3) Reject : Only to presenter who had accepted the workshop.
5) Feedback button should be display only to presenter who has accepted workshop and members of organisation who requested workshop and display on after the end date of workshop.
   Ex: if workshop is completed on 3-Nov , we should display feedback button from 4-Nov.
5) Accepted workshop to all login user if workshop is in requested status.

### PR DESCRIPTION
**Pull request by curioswati**
fixed #161

### CODE REVIEW

File: wye/workshops/templatetags/workshop_action_button.py [hunk 1]
```diff
@@ -8,20 +8,32 @@ def show_draft_button(workshop, user):
     if workshop.status in [WorkshopStatus.REQUESTED,
                            WorkshopStatus.ACCEPTED,
                            WorkshopStatus.DECLINED]:
-        return True
+
+        requester_members = workshop.requester.user.values()
+
+        for member in requester_members:
```
Comment (vnbang2003) [inline wye/workshops/templatetags/workshop_action_button.py]: In stead of loop you can just do this
if user in w.requester.user.all():  # list of organization users
    return True

File: wye/workshops/templatetags/workshop_action_button.py [hunk 2]
```diff
@@ -29,16 +41,45 @@ def show_accepted_button(workshop, use):
...
-        return True
+
+        requester_members = workshop.requester.user.values()
+        presenters = workshop.presenter.values()
+
+        for member in requester_members:
+            if user.username == member['username']:
+                is_requester = True
+
+        for presenter in presenters:
```
Comment (vnbang2003) [inline wye/workshops/templatetags/workshop_action_button.py]: We can replace for loop with if condition 
if user in workshop.presenter.all():

**Comment by vnbang2003** [comment]
@curioswati Can we please fix the for loop . Let make it pythonic code

**Comment by curioswati** [comment]
yes, done.

**Pull request merged**